xija/model.py

```diff
@@ -474,7 +496,7 @@ def make_mvals(self):
 
         # Stack the input dvals.  This *copies* the data values.
         self.n_preds = len(preds)
-        self.mvals = np.hstack(comp.dvals for comp in preds + unpreds)
+        self.mvals = np.hstack([comp.dvals for comp in preds + unpreds])
```

Comment:
@jzuhone - just to be sure can you confirm that passing an iterator to `hstack` should give the same as a list?  I'm a little surprised that the original was working, but want to be 100% sure there is no change in behavior here.

Reply:
I just checked this, they do give the same answer. NumPy was throwing lots of warnings about this.

Reply:
Thanks, sounds good.  We need to work on seeing warnings in tests and squashing them.